arch/arm64/src/zynq-mpsoc/Kconfig

```diff
@@ -35,6 +35,136 @@ config ZYNQ_MPSOC_UART1
 	---help---
 		Select to enable support for UART1.
 
+config ZYNQ_ENET
+	bool "Ethernet"
+	default n
+	select ARCH_HAVE_PHY
+	select ARCH_HAVE_NETDEV_STATISTICS
+
 endmenu # XILINX ZYNQ_MPSOC Peripheral Selection
 
+menu "Ethernet Configuration"
+	depends on ZYNQ_ENET
+
+choice
+	prompt "ZYNQ Ethernet Interface"
+	default ZYNQ_ENET4
+	depends on ZYNQ_ENET
+
+config ZYNQ_ENET1
+	bool "ENET 1"
+
+config ZYNQ_ENET2
+	bool "ENET 2"
+
+config ZYNQ_ENET3
+	bool "ENET 3"
+
+config ZYNQ_ENET4
+	bool "ENET 4"
+
+endchoice # ZYNQ Ethernet Interface
+
+config ZYNQ_GMAC_NRXBUFFERS
+	int "Number of RX buffers"
+	default 16
```

Comment:
Is there a upper limit for those 128 bytes blocks? If so, please include a range i.e. 1 to 32

Reply:
done! thanks for your review